packages/terra-list/src/ListItem.jsx

```diff
@@ -57,7 +57,7 @@ const ListItem = ({
     const chevron = <span className={cx('chevron')}><ChevronRight height="0.8em" width="0.8em" /></span>;
 
     return (
-      <li {...customProps} className={listItemClassNames} ref={refCallback}>
+      <li {...customProps} role="option" aria-selected={isSelected} className={listItemClassNames} ref={refCallback}>
```

Comment:
Should we only apply `role="option"` if the list is selectable or not necessarily?